src/auro-menu.js

```diff
@@ -43,6 +46,35 @@ class AuroMenu extends LitElement {
     ];
   }
 
+  firstUpdated() {
+    this.addEventListener('keydown', this.handleKeyDown);
+    this.addEventListener('mousedown', this.makeSelection);
+  }
+
+  updated(changedProperties) {
+    if (changedProperties.has('matchWord')) {
+      this.markOptions();
+    }
+  }
+
+  /**
+   * @private
+   * @returns {void} When called will update the DOM with visible suggest text matches.
+   */
+  markOptions() {
+    const markInstance = new Mark(this.items); // eslint-disable-line
+
+    markInstance.unmark();
+
+    if (this.matchWord && this.matchWord.length > 0) {
+      markInstance.mark(this.matchWord, {
+        'element': 'strong',
```

Comment:
So, this won't necessarily work. We have do not have absolute control over the styling of the `strong` tag. There is a default definition for the demo page for really when a markdown doc makes something `bold`, but that's about as far as it goes. 

We should manage this style specifically within the scope of the element. 

```scss
:host::slotted(strong) {}
```